napari_clusters_plotter/_tsne.py

```diff
@@ -0,0 +1,170 @@
+import pandas as pd
+import warnings
+import napari
+from qtpy.QtWidgets import QWidget, QPushButton, QLabel, QHBoxLayout, QVBoxLayout
+from qtpy.QtWidgets import QListWidget, QListWidgetItem, QAbstractItemView, QComboBox, QSpinBox
+from qtpy.QtCore import QRect
+from napari_tools_menu import  register_dock_widget
+
+@register_dock_widget(menu="Measurement > TSNE dimensionality reduction (ncp)")
+
+class TSNEWidget(QWidget):
+
+    def __init__(self, napari_viewer):
+        super().__init__()
+
+        self.viewer = napari_viewer
+
+        napari_viewer.layers.selection.events.changed.connect(self._on_selection)
+
+        self.current_annotation = None
+
+        # QVBoxLayout - lines up widgets vertically
+        self.setLayout(QVBoxLayout())
+        label_container = QWidget()
+        label_container.setLayout(QVBoxLayout())
+        label_container.layout().addWidget(QLabel("<b>Dimensionality reduction: TSNE</b>"))
```

Comment:
```suggestion
        label_container.layout().addWidget(QLabel("<b>Dimensionality reduction: t-SNE</b>"))
```
I think the algorithm is spelled like that, right?